test/connection-meta.test.js

```diff
@@ -49,6 +60,57 @@ describe("connection-meta", function() {
     });
   });
 
+  /**
+   *
+   */
+  describe("connection-batch-meta", function () {
+    this.timeout(40000); // set timeout to 40 sec.
+
+    var conn = testEnv.createConnection();
+
+    /**
+     *
+     */
+    before(function (done) {
+      this.timeout(600000); // set timeout to 10 min.
+      testEnv.establishConnection(conn, done);
+    });
+
+    /**
+     *
+     */
+    describe("describe Account, Contact", function () {
+      it("should return metadata information", function (done) {
```

Comment:
Is it possible to test the greater than 25 case?

Reply:
Yes. See the `test(conn): add test for >25 types` commit.